README.md

```diff
@@ -100,18 +101,19 @@ As of 06/11/2017, this project is in the "working prototype" phase of developmen
 
 ## Troubleshooting
 
-- While ssh'd onto the raspberry pi
+1.  Installation
+- While ssh'd onto the raspberry pi - logs should print to the screen
```

Comment:
yea this is another spot that I wrote previously that assumes you are deving on another box.